api/V1/Recording.js

```diff
@@ -0,0 +1,201 @@
+var models = require('../../models');
+var util = require('./util');
+var passport = require('passport');
+var log = require('../../logging');
+var tagsUtil = require('./tagsUtil');
+var requestUtil = require('./requestUtil');
+var responseUtil = require('./responseUtil');
+var multiparty = require('multiparty');
+var config = require('../../config/config')
+var randomstring = require("randomstring");
+var AWS = require('aws-sdk');
+
+module.exports = (app, baseUrl) => {
+  var apiUrl = baseUrl + '/recordings';
+
+  /**
+   * @api {post} /api/v1/recordings Add a new recording.
+   * @apiName PostRecording
+   * @apiGroup Recordings
+   * @apiDescription This call is the new way to upload any type of recording.
+   * It takes a 'data' field which contains a JSON object string that can
+   * contain any of the following fields, Note that 'type' is required:
+   * - type: REQUIRED, one of ('thermalRaw160x120')
```

Comment:
I wonder if including the resolution in type too specific. `thermalRaw` might be enough. The upload itself could include the frame dimensions (or it could be in `additionalMetadata`).

Reply:
Yeah, that would be better. I guess we don't want too many different "types".